### PR TITLE
Adds class method to Configuration for getting static dir

### PR DESCRIPTION
**Pull request by nballenger**
As part of the Docker refactor, I'm making a consolidated place for static files.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -466,6 +466,37 @@ def database_url(cls):
...
 
+    @classmethod
+    def static_resources_dir(cls):
+        """
+        Locate the static resources for this installation.
+
+        Default location is /simplye_static. To use a different location, set the value
+        of the SIMPLIFIED_STATIC_DIR environment variable.
+        """
+        default_static_dir = '/simplye_static'
```
Comment (leonardr) [inline config.py]: Very minor thing but it's branding related so I have to be picky -- call this `simplified_static` instead. "SimplyE" is the name of a mobile app, not the generic term that should be used here.

**Review by leonardr — APPROVED** [review]
This makes sense -- what sort of static files are we talking about? The crontab?

**Comment by nballenger** [comment]
It's the admin app js and css, font files, etc. Also the files in `resources` and `resources/images`.

File: config.py (hunk 1 above)
Comment (nballenger) [reply config.py]: Good catch. I'll go through and change it in the various places I've used it.

**Pull request merged**